test/gyx_gym_spaces_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Gyx.Gym.Spaces.Test do
```

Comment:
I think the convention is to have a folder structure similar to what you have inside your app, and try to have one test file per business logic file..
so, this file should be `tests/Gym/space_handler_test.ex` or `tests/lib/Gym/space_handler_test.ex` and have a module name like `Gyx.Gym.SpaceHandlerTest`..
it's also weird to have the module name be different that the file name.. either name the file `utils.ex`, or the module `Gyx.Gym.SpaceHandler`